src/modules/75-cf/strings/strings.en.yaml

```
@@ -330,7 +330,7 @@ environments:
     message: 'Secrets are only visible right after creation and redacted once you leave the page. Please make sure to copy and store your secret somewhere safe'
     title: 'SDK Keys'
     emptyName: 'SDK Key name is required'
-    addKey: '+ Add Key'
+    addKey: '+ Add New SDK Key'
```

You can drop the `Add` from here
```suggestion
    addKey: '+ New SDK Key'
```

---

Thanks Kevin, all done